Documentation/ReleaseNotes.md

```diff
@@ -36,10 +37,189 @@ If importing the [Mixed Reality Toolkit NuGet packages](MRTKNuGetPackage.md), th
+1. Close and re-open the project in Unity
+
 ### What's new in 2.3.0
 
+**Support for Unity 2019.3 new XR platform (Experimental)**
+
+MRTK has added initial support for Unity 2019.3's new XR platform. When using the Windows XR plugin, it is recommended using version **2.0.4 (preview.3)** or newer.
+
+Please see [Known issues](#known-issues-in-230) details on known limitations.
+
+**Better handing of compiling when using dependencies**
+
+This version of MRTK improves upon the support added in 2.2.0 to recognize and enable compilation when dependencies (ex: DotNetWinRT plugin) are required.
+
+Support has been added to automatically add or remove preprocessor defines based on the presence of the required packages and or components. This allows application projects to be more resilient to adding and removing packages. 
+
+**Common MixedRealityToolkit.Plugins folder**
+
+Previous versions of MRTK would place plugins acquired via MSBuild for Unity to a directory co-located with the .csproj file used to specify them. This version updates the .csproj file(s) to place the plugins in a shared MixedRealityToolkit.Plugins folder.
+
+This change helps address issues such as [#6972](https://github.com/microsoft/MixedRealityToolkit-Unity/issues/6972).
+
+**Hand physics extension service**
+
+A hand physics extension service has been added to allow for using physics interactions with the HoloLens 2 articulated hands ([#6573](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/6573)).
+
+![Hand physics](https://user-images.githubusercontent.com/1186832/68795768-77efdc00-0606-11ea-8fb9-b0e4191bdb05.gif)
+
+**Pinch Slider orientation**
+
+The Pinch Slider has been updated to orient TrackVisuals, TickMarks and ThumbRoot based on the sliderAxis orientation ([#6858](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/6858))
+
+![Y axis slier](https://user-images.githubusercontent.com/42405657/71687606-37a31380-2d96-11ea-84b5-ffe2368f8b57.JPG)
+![X axis slider](https://user-images.githubusercontent.com/42405657/71687640-4984b680-2d96-11ea-9f59-7732a91edd1b.JPG)
+
+**UX control prefabs updated to use PressableButton**
+
+The following prefabs are now using the PressableButton component instead of TouchHandler for near interaction ([7070](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/7070))
+
+- AnimationButton
+- Button
+- ButtonHoloLens1
+- ButtonHoloLens1Toggle
+- CheckBox
+- RadialSet
+- ToggleButton
+- ToggleSwitch
+- UnityUIButton
+- UnityUICheckboxButton
+- UnityUIRadialButton
+- UnityUIToggleButton
+
+![Updated UX prefabs](https://user-images.githubusercontent.com/36998103/72457296-30303100-37be-11ea-827f-d144a65348ff.gif)
+
 ### Known issues in 2.3.0
 
+**Issues with the Unity 2019.3 new XR platform on Windows Mixed Reality**
+
+The following issues are known when using the new XR platform and version **2.0.4 (preview.3)** of the Windows XR Plugin:
+
+- AirTap does not work on HoloLens (HoloLens 2 and 1st generation)
```

Comment:
I think that grab also doesn't work right? So basically only touch works

Reply:
@keveleigh confirmed that grab works with the specified version of the plugin.